cdap-ui/app/cdap/components/DataPrepConnections/DataPrepConnections.scss

```diff
@@ -142,6 +143,56 @@ $gutter-width: 15px;
   .connections-content {
     width: calc(100% - 250px);
 
+    .sub-panel {
+      padding: 10px;
+      display: flex;
+      .path-container,
+      .info-container {
+        width: 50%;
+      }
+      .path-container {
+        padding-right: 10px;
+        .file-path-container {
+          height: 100%;
+          .paths {
+            display: inline-flex;
+            align-items: center;
+            height: inherit;
+            width: 100%;
+            overflow: hidden;
+
+            a.active-directory {
+              width: 100%;
+              overflow: hidden;
+              text-overflow: ellipsis;
+            }
+          }
+          .collapsed-paths {
+            display: flex;
+            height: 100%;
+            align-items: center;
+          }
+        }
+      }
+      .info-container {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        padding-left: 10px;
+        justify-content: flex-end;
+        .info {
+          flex: 0.7;
+          font-size: 12px;
+          color: $info-font-color;
+          display: flex;
```

Comment:
does the children need display flex as well?

Reply:
yeah for them to right align and be next to each other.

Reply:
I think you can achieve that by simply doing class `text-xs-right` instead of using display flex and justify content

Reply:
Instead of giving 100% width to the span beneath it and making text-align right to the span I used this. But do you see anything better with the text-align right? I felt its just the implementation differences.

Reply:
I don't think you need to put width 100% width to the span for the text-align right to work. You should text-align right to here.

The reason for this suggestion, there used to be problems with nested flex display in safari. Not sure if they have fixed it by now or not. That was the problem we encounter with the market entities not having the correct sizing in safari.

Reply:
I didn't notice any in safari or ff but nevertheless have modified to change it to use text-align. Seemed simpler. 👍